docs/modules/ROOT/pages/partials/con_hermetic_troubleshooting.adoc

```diff
@@ -0,0 +1,12 @@
+.Troubleshooting
+
+ifdef::troubleshooting_builds[]
+
+If your build fails, be sure to look at your logs. Here are a few ways to find them:
+
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check, then from the resulting *Pipeline run details* view, select the *Logs* tab.
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check. In the resulting *Pipeline run details* view, click *build-container*. When the right panel opens, select the *Logs* tab to see a partial view of the log for that build.
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check, then from the resulting *Pipeline run details* view, scroll until you see *Vulnerabilities scan*. Under that, click *View logs*.
```

Comment:
Clicking `View Logs` from the Vulnerability scan section would take them to the logs tab, but auto-focus the clair-scan task, instead of build-container or pre-fetch dependencies.

Reply:
@mattreid so remove the 3rd bullet or reword?

Reply:
Sorry, I would drop that third bullet. If the pipeline fails there's a `Log snippet` section, and under that is a `See logs` link, maybe you were looking for that in a pipeline run that succeeded and found a logs link under vulnerability instead? It doesn't make a lot of sense to me to recommend a pathway for hermetic that is tied to vulnerability scanning, even if it does takes them to the logs tab. They may as well just have followed bullet 1.